internal/deviations/deviations.go

```diff
@@ -208,6 +208,16 @@ var (
 	NetworkInstanceTableDeletionRequired = flag.Bool("deviation_network_instance_table_deletion_required", false,
 		"Set to true for device requiring explicit deletion of network-instance table, default is false")
 
+	SwitchChipIDUnsupported = flag.Bool("deviation_switch_chip_id_unsupported", false, "Device does not support id leaf for SwitchChip components. Set this flag to skip checking the leaf.")
```

Comment:
Please use this new format for introducing new deviations: https://github.com/openconfig/featureprofiles/pull/1358.

We will be adding a check for this soon, all new flags should not be exported (define them in lowercase) and accessed from tests through a public accessors (ref in PR 1358)

Reply:
Done